templates/definition/defaults.yaml

```diff
@@ -224,6 +224,26 @@ params:
       en: "'en' for English and 'de' for German"
     default: en
     validvalues: ["de", "en"]
+  - name: icon
```

Comment:
In die advanced config so wie bei offline auch?

Reply:
Das sollte bereits ne Advanced Config sein. Siehe `vehiclebase` in `defaults.yaml`. Ich hatte allerdings übersehen, dass nicht alle Fahrzeuge die `vehiclebase` nutzen (Tesla, Tronity, ...). Das ist jetzt nachgezogen.

Reply:
Yep. base haben nur die mit User/Passwort